src/components/ui4/DotsBackground.tsx

```diff
@@ -9,11 +9,15 @@ import { useTheme } from '../services/ThemeContext'
 
 export type UpdateDots = Array<Partial<ThemeDot> | undefined | null>
 interface Props {
+  // Optional backgroundGradient overrides
+  backgroundGradientColors?: string[]
+  backgroundGradientStart?: { x: number; y: number }
+  backgroundGradientEnd?: { x: number; y: number }
```

Comment:
If `ThemeGradient` was a type, then it could be one field:

```ts
gradient?: ThemeGradient
```

Reply:
`backgroundGradient` is fine too. What I'm golfing for is keeping these values together under one type.